### PR TITLE
Update Set-CsTeamsCallingPolicy.md

### PR DESCRIPTION
**Pull request by nitinjaitely**
Making the changes to the note for clarity.

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md [hunk 1]
```diff
@@ -261,7 +261,7 @@ Accept wildcard characters: False
 ```
 
 ### -BusyOnBusyEnabledType
-Setting this parameter lets you configure how incoming calls are handled when a user is already in a call or conference or has a call placed on hold. Valid options are: Enabled, Unanswered, and Disabled. When set to Enabled, new or incoming calls will be rejected with a busy signal. When set to Unanswered, the user's unanswered settings will take effect, such as routing to voicemail or forwarding to another user. Note: The UserOverride option value is not available for use currently, if set it will be read as setting the value to Disabled.
+Setting this parameter lets you configure how incoming calls are handled when a user is already in a call or conference or has a call placed on hold. Valid options are: Enabled, Unanswered, and Disabled. When set to Enabled, new or incoming calls will be rejected with a busy signal. When set to Unanswered, the user's unanswered settings will take effect, such as routing to voicemail or forwarding to another user. Note: UserOverride setting is not currently available for use. As of now, this setting will be treated as disabled even if its is configured.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md:264]:
```suggestion
Setting this parameter lets you configure how incoming calls are handled when a user is already in a call or conference or has a call placed on hold. Valid options are: Enabled, Unanswered, and Disabled. When set to Enabled, new or incoming calls will be rejected with a busy signal. When set to Unanswered, the user's unanswered settings will take effect, such as routing to voicemail or forwarding to another user. Note: The userOverride setting is not currently available for use. The setting will be treated as disabled even if it is configured.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @nitinjaitely, kindly commit the suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@nitinjaitely Gentle reminder.

**Comment by yogkumgit** [comment]
@nitinjaitely kindly implement the suggested changes, else we may proceed to close this PR. Thanks

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws

**Comment by yogkumgit** [comment]
@nitinjaitely Closing as it is same as #10460

**Pull request closed without merging**